collect_app/src/main/java/org/odk/collect/android/dto/Instance.java

```diff
@@ -157,15 +149,27 @@ public Long getLastStatusChangeDate() {
         return lastStatusChangeDate;
     }
 
-    public String getDisplaySubtext() {
-        return displaySubtext;
-    }
-
     public Long getDeletedDate() {
         return deletedDate;
     }
 
     public Long getDatabaseId() {
         return databaseId;
     }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this || other instanceof Instance
```

Comment:
How did you pick which subset of fields to include here? For example, why not all fields? Why display name and not version? Why not only instance file path?

Reply:
We don't have instanceId column but yeah I can just use instance file path. FIxed.